deploy/operator/apply/01_FabricGatewayCRD.yaml

```diff
@@ -82,6 +82,33 @@ spec:
                   will be able to access the API even if they match all other authorization requirements. If the whitelist
                   is empty, then no service tokens can access via the gateway
                 example: stups_some-service-name
+            x-fabric-cors-support:
+              type: object
+              description: |
+                This section configures Cross-Origin Resource Sharing (CORS) support in the gateway. An HTTP OPTIONS endpoint will be created for each path,
+                fulfilling the preflight requirement for CORS. Currently the preflight request is the only part of CORS we support, we do not add the allowed origins
```

Comment:
I don't think you need to include info on what Gateway doesn't do, wrt to not mutating responses. State what it does only

Reply:
Reworded it as a note as opposed to saying what is not done. I think it's good to call it out